.gitattributes

```
@@ -0,0 +1 @@
+Cargo.lock text eol=lf linguist-generated=false
```

```suggestion
Cargo.lock linguist-generated=false
```
to match what is done in the [Rust repo](https://github.com/rust-lang/rust/blob/6e23095adf9209614a45f7f75fea36dad7b92afb/.gitattributes#L12)